### PR TITLE
PIA-3208: fix(GiniCaptureSDK): fix flash button alignment

### PR DESCRIPTION
**Pull request by chriskryn**
PIA-3208

### CODE REVIEW

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Looks good when all configuration options are switched on:
- when flash is disabled (could you please check with Anne if it's ok)
IMG_305457118343-1
- when the import is disabled the import button should be hidden, but it's still presented.

https://user-images.githubusercontent.com/3911816/197787969-31d0eae1-15b4-4c44-a759-7234071883e1.MP4

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift [hunk 1]
```diff
@@ -103,15 +105,60 @@ final class CameraPane: UIView {
...
+            number -= 1
+        }
+        if fileUploadButton.isHidden {
+            number -= 1
+        }
+        return number
+    }
+
+    private func onlyLeftVisibleButtonImgWidth() -> CGFloat {
+        if flashButton.isHidden == false {
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift:136]: [Suggestion] `if !flashButton.isHidden`

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift [hunk 2]
```diff
@@ -103,15 +105,60 @@ final class CameraPane: UIView {
...
+            fileUploadButton.isHidden = isHidden
+        }
         if thumbnailView.thumbnailImageView.image != nil {
             thumbnailView.isHidden = isHidden
         }
+        updateThumbnailConstraint()
+    }
+
+    private func numberOfVisibleButtons() -> Int {
+        var number = 2
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift:125]: [Suggestion] Maybe it is rocket science, but: `[flashButton.isHidden, fileUploadButton.isHidden].filter({$0}).count`

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift [hunk 3]
```diff
@@ -103,15 +105,60 @@ final class CameraPane: UIView {
...
+
+    private func onlyLeftVisibleButtonImgWidth() -> CGFloat {
+        if flashButton.isHidden == false {
+            return flashButton.iconView.image?.size.width ?? 0
+        }
+        return fileUploadButton.iconView.image?.size.width ?? 0
+    }
+
+    func updateThumbnailConstraint(
+    ) {
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift:143]: Is this newline needed here?

**Review by zladzeyka — APPROVED** [review]
Looks good 😊👏🏻🎉

**Review by vizaknaidavid — APPROVED** [review]

**Review by zladzeyka — CHANGES_REQUESTED** [review]
IMG_0020
There's a tiny issue on the iPad with a landscape orientation. 🥲

**Review by zladzeyka — APPROVED** [review]
Looks good!

**Pull request merged**